tensorflow/contrib/go/proto/allocation_description.pb.go

```diff
@@ -0,0 +1,129 @@
+// Code generated by protoc-gen-go.
```

Comment:
Drive-by comment 2: all other languages don't require checking-in the product of the protoc-generated code.  I am hoping this can be auto-generated like for python and C++ -- can you make sure this is possible?